recipes/cuda-kat/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(cuda-kat REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME} test_package.cpp)
+target_link_libraries(${PROJECT_NAME} cuda-kat::cuda-kat)
+set_target_properties(${PROJECT_NAME} PROPERTIES CXX_STANDARD 17)
```

Comment:
Is C++ really required? If yes, need to update the recipe to validate the used compiler.

Reply:
Thank you for your feedback.

I accepted all your suggestions and added validation for compiler supported version.